contrib/grimshot

```diff
@@ -32,13 +32,14 @@ FILE=${3:-$(getTargetDirectory)/$(date -Ins).png}
 
 if [ "$ACTION" != "save" ] && [ "$ACTION" != "copy" ] && [ "$ACTION" != "check" ]; then
   echo "Usage:"
-  echo "  grimshot [--notify] (copy|save) [active|screen|output|area|window] [FILE]"
+  echo "  grimshot [--notify] (copy|save) [active|screen|output|area|window] [FILE|-]"
   echo "  grimshot check"
   echo "  grimshot usage"
   echo ""
   echo "Commands:"
   echo "  copy: Copy the screenshot data into the clipboard."
   echo "  save: Save the screenshot to a regular file."
```

Comment:
This line should be removed, right?

Reply:
This change helps document this special case. Would you like me to remove it?

Reply:
I mean, it's the same as the one below, except with less details.

Reply:
That is why the line exists-- to be a one-line summary of the usage. 

Would you like me to remove the change?

Reply:
The summary for "save" appears twice. Delete one, keep the other.

Reply:
Sorry for the confusion. Your comment appeared attached to the to the "usage" line rather than the duplicated line. I've force -pushed a removal of the duplicated line. 

Thanks for the rapid feedback.